CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,84 @@
+# Code of Conduct
+
+## 1. Purpose
+
+A primary goal of C Riva.GitHub.Io is to be inclusive to the largest number of contributors, with the most varied and diverse backgrounds possible. As such, we are committed to providing a friendly, safe and welcoming environment for all, regardless of gender, sexual orientation, ability, ethnicity, socioeconomic status, and religion (or lack thereof).
```

Comment:
Any reason why you put `C Riva.GitHub.Io` rather than `criva.github.io`?

Reply:
It's because he's a doof.